apps/ledger-live-desktop/src/renderer/screens/manager/FirmwareUpdate/index.tsx

```diff
@@ -95,7 +95,9 @@ const FirmwareUpdate = (props: Props) => {
       firmwareName: firmware.final.name,
     });
 
-    setFirmwareUpdateOpened(true); // Prevents manager from reacting to device changes (?)
+    // Prevents manager from reacting to device changes
+    setPreventResetOnDeviceChange(true);
```

Comment:
same here, `setPreventResetOnDeviceChange(false)` should probably be set to false when the drawer closes ?

Reply:
Done 👍